### ISSUE TITLE
Product Variations: Create the Bulk update screen

### ISSUE THREAD

**Issue by chrisKns** [woocommerce/woocommerce-ios#6239]
This is part of the Bulk update prices for Product Variations project p91TBi-7x3-p2. See parent issue #5762. This issue has the sub-tasks for creating the a new bulk update screen that the user can select that value to update:
Implement after #6238

<img src="https://user-images.githubusercontent.com/562080/152804618-e176a331-bb3e-47bf-8922-f121499dd766.png" width=20% height=20%>

We will create a "bulk update" screen to allow a user to select what value(currently the price only) he will update for all the product variations. For each value he will be have a text informing him if all the variations have the same value or not. Taping on price value he will be shown a "bulk price setting" screen.

Sub-tasks:

- [x] Create view model that providing sale & regular price formatted as string value (for none, mixed, or common). #6341
- [x] Make sure “Bulk Update Screen” loads all variations #6315
- [x] Show redacted screen while variations are loaded #6315
- [x] Add to the view model handling the "cancel" event. #6361
- [x] Add support for “disabled” text colour for the "None" displaying value text . #6362
- [x] Create new "Bulk update Screen" showing sale and regular price rows and section #6315
- [x] Handle "cancel" button #6361
- [x] modally present "Bulk update Screen"  on tapping the “bulk update CTA” #6315
- [x] Handle tapping on bulk edit regular & sale price, pushing/poping the "bulk price setting screen",  along with presenting success message on bulk price update #6403
- [x] Handle case where variation synchronisation fails. Display a `Notice` with a retry button #6375
- [x] Do not show option to present "Bulk update Screen" if product has no variation (moved to separate issue #6340)
- [ ] Fix "jump" and changes from the loading state to the loaded state #6376
